### ISSUE TITLE
symbolic link issue

### ISSUE THREAD

**Issue by Unlimitosu** [jasmin-lang/jasmin#566]
I built jasmin but `jasminc` symbolic link doesn't work.

I did the following:
1. Clone the jasmin repository
2. run `make` at `jasmin/proofs`
3. run `make` at `jasmin/compiler`

Then I could find `jasminc` in `jasmin/compiler` like below.
```
unlimit@ubuntu:~/Documents/constant-time/jasmin/compiler$ ls
_build  default.nix   entry     jasmin.opam  Makefile   safety     src
CCT     dune          examples  jazz2tex     MANIFEST   safetylib  syscall
config  dune-project  jasminc   LICENSE      README.md  scripts    tests
```

And I could find symbolic link like below.
```
unlimit@ubuntu:~/Documents/constant-time/jasmin/compiler$ ll
total 88
    ...
-rw-rw-r--  1 unlimit unlimit   90 Sep 12 02:49 .gitignore
lrwxrwxrwx  1 unlimit unlimit   32 Sep 12 20:48 jasminc -> _build/default/entry/jasminc.exe*
-rw-rw-r--  1 unlimit unlimit  976 Sep 12 02:49 jasmin.opam
lrwxrwxrwx  1 unlimit unlimit   33 Sep 12 20:48 jazz2tex -> _build/default/entry/jazz2tex.exe*
     ...
```

However, if I run `jasminc <filename>` at other directory, the command `jasminc` cannot be found.

How can I solve this problem?

**Comment by vbgl** [comment]
This is the expected behavior. What are you trying to do?

If you just want to run the jasmin compiler, follow the instructions given on the first lines of that page: https://github.com/jasmin-lang/jasmin/wiki/Installation-instructions.

**Comment by eponier** [comment]
I think @Unlimitosu wants to install the binaries. `make install` should do the job. Note that if you don't want to develop Jasmin, just to use it, you have simpler ways to install it using nix or opam.

**Comment by Unlimitosu** [comment]
Yes, @eponier said what I mean. `make install` works well and now I can use Jasmin. Thanks a lot!
